x-pack/plugins/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -69,10 +76,17 @@ function getCurrentTab(
   return selectedTab ? selectedTab : first(tabs) || {};
 }
 
-export function DetailView({ errorGroup, urlParams, kuery }: Props) {
+export function ErrorSampleDetails({
+  onSampleClick,
+  errorSamples,
+  errorSamplesFetchStatus,
+  errorData,
+  errorFetchStatus,
+  occurrencesCount,
+}: Props) {
+  const [sampleActivePage, setSampleActivePage] = useState(0);
   const history = useHistory();
-  const { transaction, error, occurrencesCount } = errorGroup;
-
+  const { urlParams } = useLegacyUrlParams();
   const { detailTab, offset, comparisonEnabled } = urlParams;
```

Comment:
[:nit] Since `urlParams` is only used for destructuring, maybe we can do something like:
```
const { urlParams: {detailTab, offset, comparisonEnabled} } = useLegacyUrlParams();
```